docs/intro.md

```diff
@@ -1,20 +1,12 @@
 ---
-title: "FAQ"
-linkTitle: "FAQ"
-weight: 3
-categories: ["quickstart", "uor"]
-tags: ["edu","docs"]
-description: >
-  Frequently Asked Questions
+sidebar_position: 1
 ---
 
-== Universal Object Reference FAQ
-:toc:
-:toclevels: 3
-:sectnumlevels: 3
+# FAQ
 
+Let's cover some basics of UOR.
 
-### What is UOR? 
+## What is UOR?
 
 Universal Object Reference (UOR) provides a single format to publish, retrieve, and interact with any content. By “any content” we mean literally anything.
```

Comment:
`UOR is a framework used to generalize multiple streams of content management into a single content management mechanism and strategy. `

Reply:
resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)